pkgs/build-support/dotnet/add-nuget-deps/default.nix

```diff
@@ -4,12 +4,10 @@
   nix,
   lib,
   substituteAll,
-  nuget-to-nix,
-  nixfmt-rfc-style,
-  nuget-to-json,
   cacert,
   fetchNupkg,
   callPackage,
+  buildPackages,
```

Comment:
Sorry for the dumb question but what's `buildPackages`?
Never saw it being used before

Reply:
Packages for the build system platform as opposed to the host/target system.
It's where `nativeBuildInputs` gets the packages through splicing magic (which is why it's so confusing)
More info here https://nixos.org/manual/nixpkgs/unstable/#sec-cross-infra